machine-learning/lambda-v2/README.md

```diff
@@ -2,14 +2,20 @@
 
 These example components are used to run sample image classification and object detection inferences with AWS lambda.
 
+
 Image Classification 
- - com.lambda.DLRImageClassification 
```

Comment:
The `com.lambda` prefix suggests that these samples are created by the Lambda team, which isn't accurate. How about one of the following prefixes?

* `aws.greengrass.sample` to indicate that this is a sample provided by AWS IoT Greengrass.
* `aws.greengrass.sample.lambda` to indicate that this is a Lambda sample provided by AWS IoT Greengrass.
* `aws.greengrass.labs` to align with community components.
* `aws.greengrass.labs.lambda` to align with community components and indicate that this is a Lambda component.

Reply:
Changing name here would result in changing file names of the components  (recipe and artifact file names) making this a big PR with just more than readme changes. 
Moreover, sagemaker edge manager components are using com.* notation following other examples. These are not aws.* because these are not 1p or community components. I'm more inclined to leaving it this way but lmk what you think about this and in that case, I'd prefer `aws.greengrass.sample.lambda`:)

Reply:
I've renamed the files as suggested.